components/Joblistings/JoblistingsContainer.tsx

```diff
@@ -80,22 +82,37 @@ const CompanyElement = styled('div')`
   }
 `;
 
+/**
+ * For some reason, a node in a joblistingconnection can be null, and TS is not smart enough
+ * to know that we filter it from the array, so
+ * we define the type here, without `null`.
+ */
+type JoblistingNode = NonNullable<
```

Comment:
Should be able to make it non-nullable by doing something like in the api:

```python
    joblistings = OrderedDjangoFilterConnectionField(
        graphene.NonNull(Joblisting),
```

Reply:
Apparently, it's not as easy as that, but it would be nice to have both the edges and nodes nonnullable. I've seen there are a bunch of issues (and solutions) about just this, so shouldn't be too bad.